### PR TITLE
S0Or3thE change login response message

### PR DESCRIPTION
**Pull request by samuel776**
#### What does this PR do?
  - Fix bug in login
#### Description of Task to be completed?
 - appropriate error message displayed on invalid inputs
 - appropriate error message displayed on unsuccessful attempts
#### How should this be manually tested?
- checkout this branch
- run npm install
#### What are the relevant trello id?
[Or3thS0E](https://trello.com/c/Or3thS0E/8-sign-in-to-phantom)

### CODE REVIEW

**Review by mansurissa — APPROVED** [review]
LGTM

File: src/controllers/locales/en.json [hunk 1]
```diff
@@ -149,17 +149,5 @@
...
-  "Successfully deleted a user": "Successfully deleted a user",
-  "Role Driver is not allowed": "Role Driver is not allowed",
-  "Role Operator is not allowed": "Role Operator is not allowed",
-  "You are not allowed. Check Your password": "You are not allowed. Check Your password",
-  "You are not allowed to view the profile": "You are not allowed to view the profile",
-  "You are not authorized, check your Id": "You are not authorized, check your Id",
-  "you are not allowed to view the profile": "you are not allowed to view the profile",
-  "Not authorized. Not owner": "Not authorized. Not owner",
-  "Not authorized": "Not authorized",
-  "Not allowed to perform this action": "Not allowed to perform this action"
```
Comment (raymond42) [inline src/controllers/locales/en.json]: Were all these supposed to be deleted?

File: src/controllers/usersController.js [hunk 2]
```diff
@@ -185,7 +186,7 @@ export const signin = async (req, res) => {
           });
         })();
       } else {
-        return errorRes(res, 400, res.__('Incorrect password'));
+        return errorRes(res, 409, res.__('Incorrect Email or password'));
```
Comment (raymond42) [inline src/controllers/usersController.js]: why did you change the status code to 409? it should be 400

**Review by raymond42 — CHANGES_REQUESTED** [review]

**Review by GuevaraMarius — COMMENTED** [review]
Good work Samuel, the messages to correct are in English only?

File: src/controllers/locales/en.json (hunk 1 above)
Comment (samuel776) [reply src/controllers/locales/en.json]: not to be deleted, if so it was by intentionally i can fix  them

File: src/controllers/usersController.js (hunk 2 above)
Comment (samuel776) [reply src/controllers/usersController.js]: yeah sure thanks Ray, i can respond with a bad request instead of conflicts

**Review by raymond42 — APPROVED** [review]

**Review by KundwaBruno — APPROVED** [review]

**Review by raymond42 — CHANGES_REQUESTED** [review]
LGTM

**Review by raymond42 — APPROVED** [review]
